packages/vetrina/src/Vetrina/Vetrina.purs

```diff
@@ -354,8 +354,8 @@ vetrinaContainer purchaseState child =
 orderErrorMessage :: OrderFailure -> String
 orderErrorMessage failure =
   case failure of
-    AuthenticationError -> "Kombinationen av e-postadress och lösenord finns inte"
-    _                   -> "Något gick fel. Vänligen försök igen om en stund."
+    AuthenticationError _ -> "Kombinationen av e-postadress och lösenord finns inte"
```

Comment:
This could use a comment with it `-- TODO this line is actually never used` or something since apparently it is so. For future reference for whoever braves this mess again.

Reply:
Yeah, I was thinking about removing it altogether but didn't dare to. Same with "errorMessage" in NewPurchase.purs state.

Reply:
I added the comment